web/ui/react-app/src/themes/shared.scss

```diff
@@ -0,0 +1,40 @@
+input[type='checkbox']:checked + label {
+  color: $primary;
+}
+
+.tab-content {
+  border-left: 1px solid $nav-tabs-border-color;
+  border-right: 1px solid $nav-tabs-border-color;
+  border-bottom: 1px solid $nav-tabs-border-color;
+}
+
+.rule_cell {
```

Comment:
Let's use the opportunity to make the old under_scored CSS class names (`rule_cell` and `alert_cell`) consistent with the dash-separated ones :)